packages/create-yoshi-app/templates/business-manager-module/javascript/templates/module_{%PROJECT_NAME%}.json

```
@@ -0,0 +1,30 @@
+{
+  "moduleId": "{%PROJECT_NAME%}",
+  "mainPageComponentId": "{%projectName%}-lazy-component-id",
```

Copied over mistake but this should just be:
```suggestion
  "mainPageComponentId": "{%projectName%}",
```